app/views/trips/_trip.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="container-lg" style="border-style: solid; border-width: 1px; border-radius: 7px; padding: 10px 25px; margin-top: 10px">
+    <a href=<%= "/trips/" + trip.id.to_s %> style="text-decoration: none; color: black">
```

Comment:
You can use the better `link_to` here (it also works for blocks)

```erb
<%= link_to(@profile) do %>
  <strong><%= @profile.name %></strong> -- <span>Check it out!</span>
<% end %>
# => <a href="/profiles/1">
       <strong>David</strong> -- <span>Check it out!</span>
     </a>
```

---

Instead of an inline style, you can use Bootstrap's [`.text-decoration-none`](https://getbootstrap.com/docs/4.6/utilities/text/#text-decoration) and [`.text-black-50`](https://getbootstrap.com/docs/4.6/utilities/colors/) CSS classes.